prow/plugins/lifecycle/lifecycle.go

```diff
@@ -72,9 +76,9 @@ type commentClient interface {
 }
 
 type lifecycleClient interface {
-	CreateComment(owner, repo string, number int, comment string) error
 	AddLabel(owner, repo string, number int, label string) error
 	RemoveLabel(owner, repo string, number int, label string) error
+	GetIssueLabels(org, repo string, number int) ([]github.Label, error)
 }
 
 func deprecate(gc commentClient, plugin, org, repo string, number int, e *github.GenericCommentEvent) error {
```

Comment:
Don't suppose you'd be interested in deleting the deprecated stuff?

Reply:
> Don't suppose you'd be interested in deleting the deprecated stuff?

Done, deprecated stuff is now gone. :tada:

Reply:
Awesome, thanks!!